src/firefly/js/tables/TableUtil.js

```diff
@@ -120,7 +120,7 @@ export function makeIrsaCatalogRequest(title, project, catalog, params={}, optio
     title = title || catalog;
     options.use = options.use || 'catalog_overlay';
     const tbl_id = options.tbl_id || uniqueTblId();
-    const id = 'GatorQuery';
+    const id = params.processorName||'GatorQuery';
```

Comment:
Can you please undo this change?  This function is called makeIrsaCatalogRequest which is a request to the gator service at IRSA.  Passing a different 'id' will void all of the defined parameters.  There are other function you can use when you want to supply your own 'id', ie. makeTblRequest